src/Mutant/MutantInterface.php

```diff
@@ -52,5 +53,8 @@ public function getDiff(): string;
 
     public function isCoveredByTest(): bool;
 
+    /**
+     * @return CoverageLineData[]
```

Comment:
This could be extracted in a separate PR

Reply:
https://github.com/infection/infection/pull/839

Reply:
I don't think adding one doc block worth creating a separate PR